src/Parser/Mage/Fire/Modules/Cooldowns/Combustion.js

```diff
@@ -0,0 +1,38 @@
+import SPELLS from 'common/SPELLS';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import Analyzer from 'Parser/Core/Analyzer';
+import HIT_TYPES from 'Parser/Core/HIT_TYPES';
+import SpellUsable from 'Parser/Core/Modules/SpellUsable';
+
+const COMBUST_REDUCTION_SPELLS = [
+	SPELLS.FIREBALL.id,
+	SPELLS.PYROBLAST.id,
+	SPELLS.FIRE_BLAST.id,
+];
+
+class Combustion extends Analyzer {
+
+	static dependencies = {
+		combatants: Combatants,
+		spellUsable: SpellUsable,
+	}
+
+	reductionAmount = 1000
+	cooldownReduction = 0;
```

Comment:
This not currently used, but it makes sense to keep because it can be used for a future Kindling statistic.